pkg/services/ngalert/models/alertmanager.go

```diff
@@ -2,6 +2,8 @@ package models
 
 import "time"
 
+const AlertConfigurationVersion = 1
```

Comment:
```suggestion
const AlertConfigurationVersion = "v1"
```

Reply:
I'm passing this to the command before saving it:
https://github.com/grafana/grafana/blob/9bb10a7a156421e5b03fa6febac99d92305b74d2/pkg/services/ngalert/api/api_alertmanager.go#L345
Do you prefer to change `AlertConfigurationVersion` to be a string?

Reply:
Completely missed that. The current approach LGTM.